app/components/primer/markdown.rb

```diff
@@ -93,7 +93,7 @@ class Markdown < Primer::Component
     #     <ol>
     #       <li>Michael Jackson</li>
     #       <li>Michael Bolton</li>
-    #       <li>Michael Buble</li>
+    #       <li>Michael Bublé</li>
```

Comment:
I didn't see any specific reason to use the unaccented form, so this makes the entry consistent w/ another instance in this repository.

Reply:
I'm pretty sure the accented form is more correct anyway, since that's the way his name is actually spelled.